### PR TITLE
Add a deployment instruction

### PR DESCRIPTION
**Pull request by yinlinchen**
**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2415) (:star:)

# What does this Pull Request do? (:star:)
A deployment instruction for deploying dlp-access demo site. 

See preview: https://dev.d1ts8x5hxswi0x.amplifyapp.com/

# How should this be tested?
See https://github.com/VTUL/dlp-access/blob/LIBTD-2415/docs/deploy.md
This document is for a brand new deploy with all the demo records.

Please check wording and anything need to add. 

# Interested parties
Tag (@VTUL/dld-dev ) interested parties

(:star:) Required fields

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```
+aws dynamodb put-item --table-name Archive-yourtablename --item file://archive_item1.json
+```
+
+You will see a list of demo items in the `Demo collection` page
+
+<img src="images/democollectionitems.png" width="80%"/>
+
+Currently we support image, video, audio, IIIF image, PDF, podcast, X3D, and VT Kaltura Video.
```
Comment (tingtingjh) [inline docs/deploy.md:94]: Currently the file types we support are images, IIIF images, PDFs, audio(mp3, ogg, wav) , video (mp4, mov, VT Kaltura Video), X3D, etc.

**Review by tingtingjh — COMMENTED** [review]
Just some text editing.

**Comment by yinlinchen** [comment]
@tingtingjh It is updated. Thanks for the review.

**Review by tingtingjh — APPROVED** [review]
Great work on providing one-button deploy feature. I approve.

**Pull request merged**